### PR TITLE
Wouter

### PR DESCRIPTION
**Pull request by claragimenojesus**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -8,31 +8,49 @@
...
-git tag -l
-git checkout tags/t.6.0
+#git tag -l
+git checkout tags/6.0.2
 cd ..
+export PATH=$HOME/.local/cylc/bin:$PATH
 cylc --version
 cd ~/.local/cylc
-make
+make                            # WB: is install needed?? What does this do? Getting latex compilation errors
```
Comment (claragimenojesus) [inline setup.sh]: no need for make. 

To check software just need to run :
```
 cylc check-software
cd ..
```

File: setup.sh [hunk 2]
```diff
@@ -8,31 +8,49 @@
...
 cd ../..
 
-export PATH =$HOME/.local/cylc/bin:$PATH
-.~/.bashrc
+export PATH=$HOME/.local/bin:$PATH
+.~/.bashrc                      # WB: what is this for?
```
Comment (claragimenojesus) [inline setup.sh:46]: run your .bashrc file or log off and log in again to append the Cylc command set to your library

File: setup.sh [hunk 3]
```diff
@@ -8,31 +8,49 @@
...
+git checkout tags/6.0.2
 cd ..
+export PATH=$HOME/.local/cylc/bin:$PATH
 cylc --version
 cd ~/.local/cylc
-make
+make                            # WB: is install needed?? What does this do? Getting latex compilation errors
 cd ../..
 
-export PATH =$HOME/.local/cylc/bin:$PATH
```
Comment (claragimenojesus) [inline setup.sh:25]: why not in the cylc folder?

File: setup.sh [hunk 4]
```diff
@@ -8,31 +8,49 @@
...
 cd ~/.local
-git clone https://github.com/metome/rose.git
+git clone git@github.com:metomi/rose.git
 cd rose
-git tag -l
-git checkout tags/2018.02.0
+#git tag -l
+git checkout tags/2018.02.0     # Clara and Simon use more recent version (2019.01)
 cd ..
+export PATH=$HOME/.local/rose/bin:$PATH
```
Comment (claragimenojesus) [inline setup.sh:57]: add:
`chmod -R 755 ~/.local`

then need to run .~./bashrc or log off and in again to export the PATH

File: setup.sh [hunk 5]
```diff
@@ -53,20 +71,26 @@ prefix-location.u=https://code.metoffice.gov.uk/svn/roses-u
 prefix-web.u=https://code.metoffice.gov.uk/trac/roses-u/intertrac/source:
 prefix-ws.u=https://code.metoffice.gov.uk/rosie/u" >> rose.conf
 
-export PATH=$HOME/.local/rose/bin:$PATH
```
Comment (claragimenojesus) [inline setup.sh:56]: JULES instructions suggest best to append the Rose command after generating the above text file

File: setup.sh [hunk 6]
```diff
@@ -77,14 +101,13 @@ metofficesharedrepos =code*.metoffice.gov.uk
 username =yourusername
 store-plaintext-passwords=no" >> servers.txt
 
-export PATH =$HOME/.local/fcm/bin:$PATH
```
Comment (claragimenojesus) [inline setup.sh:80]: best here? 
add
`chmod -R 755 ~/.local`

File: setup.sh [hunk 7]
```diff
@@ -95,35 +118,21 @@ browser.comp-pat[jules_doc.x] = (?msx-i:\A // [^/]+ /svn/ ([^/]+) /*(.*) \z)" >>
 
 ## Install JULES
```
Comment (claragimenojesus) [inline setup.sh:151]: **Last step missing: Caching your MOSRS password**

#Caching your MOSRS password may be problematic if your MOSRS password has special characters in it (eg "%","!" or "&")

#Download two MOSRS utilities from [here]# (https://code.metoffice.gov.uk/trac/home/wiki/AuthenticationCaching/GpgAgent):

   #firefox https://code.metoffice.gov.uk/trac/home/raw-attachment/wiki/AuthenticationCaching/GpgAgent/mosrs-setup-gpg-agent

#Put in your MOSRS password, save the file and close Firefox.

   #firefox https://code.metoffice.gov.uk/trac/home/raw-attachment/wiki/AuthenticationCaching/GpgAgent/mosrs-cache-password

#Again, put in your MOSRS password, save the file and close Firefox.

```
 mv ~/Downloads/mosrs-setup-gpg-agent ~/.local/mosrs-setup-gpg-agent

   mv ~/Downloads/mosrs-cache-password ~/.local/mosrs-cache-password

   export PATH=$HOME/.local:$PATH

   chmod -R 755 ~/.local
```

#Now activate
`   . mosrs-setup-gpg-agent`

#To test this has worked correctly
`rosie hello`
#svn info --username=[yourusername] https://code.metoffice.gov.uk/svn/test

#you will need to re-cache your MOSRS password every time you log in from now on.
`export PATH=$HOME/.local:$PATH`
`[[ "$-" != *i* ]] && return`
`. mosrs-setup-gpg-agent`

File: setup.sh [hunk 8]
```diff
@@ -95,35 +118,21 @@ browser.comp-pat[jules_doc.x] = (?msx-i:\A // [^/]+ /svn/ ([^/]+) /*(.*) \z)" >>
...
 export JULES_ROOT=$PWD
 echo $JULES_ROOT
 
+# note: JULES already comes with make.cfg. Why does this need to be overwritten?
+
 cp /home/clara/make.cfg ~/MODELS/jules-vn6.1/etc/fcm_make
 cd $JULES_ROOT
 fcm make -j 2 -f etc/fcm-make/make.cfg --new
 
-## Set up Anaconda on your home account
```
Comment (claragimenojesus) [inline setup.sh:139]: why is this not needed?

File: setup.sh (hunk 1 above)
Comment (ICHydro) [reply setup.sh]: ok, thanks!

File: setup.sh (hunk 3 above)
Comment (ICHydro) [reply setup.sh:25]: Should be there in line 38?

File: setup.sh (hunk 5 above)
Comment (ICHydro) [reply setup.sh:56]: ok!

File: setup.sh (hunk 8 above)
Comment (ICHydro) [reply setup.sh:139]: replaced by lines 13 - 15

**Pull request merged**